### PR TITLE
Adaptive Learning

### LINKED ISSUE #108: Knowledge Distillation for Smart Distancing
I have an idea to use knowledge distillation for each new environment and and train an environment specific models.
@mhejrati what is your idea?

### PR DESCRIPTION
**Pull request by alpha-carinae29**
This PR will add bootstrapping feature to smart distancing app.
by using a strong and heavy teacher model we create labels and train a lightweight model (student model) based on that label in each training round.
This will close #108

### CODE REVIEW

**Comment by JsonSadler** [comment]
Wow great work. 
What happened to this PR? 
@alpha-carinae29  Are you going to make another PR to the new Repo?

File: applications/adaptive-learning/configs/config_engine.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python3
+import configparser
+import threading
+
+
+class ConfigEngine:
```
Comment (JsonSadler) [inline applications/adaptive-learning/configs/config_engine.py:6]: Later you can use the latest version of config parser from the new repo containing the smart-social-distancing code.

File: applications/adaptive-learning/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    build:
+      context: ./students/
+      dockerfile: ssd-mobilenet-v2-student.Dockerfile
+    stdin_open: true
+    tty: true
+    container_name: student-container
+    volumes:
+    - ../../:/repo
+    ports:
+      - "2029:6006"
```
Comment (JsonSadler) [inline applications/adaptive-learning/docker-compose.yml:27]: Better mention this default port in README

File: applications/adaptive-learning/students/ssd-mobilenet-v2-student.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#This container will install TensorFlow Object Detection API and its dependencies in the /model/research/object_detection directory
+
+FROM tensorflow/tensorflow:1.15.0-gpu-py3
```
Comment (JsonSadler) [inline applications/adaptive-learning/students/ssd-mobilenet-v2-student.Dockerfile:3]: as this code is placed in neuralet repo, I suggest mentioning the supported hardware platform (e.g. amd64 with Nvidia GPU) in the README.

**Review by JsonSadler — COMMENTED** [review]
Great work, thanks :+1:

**Review by mhejrati — APPROVED** [review]
Great work, thanks.

**Pull request merged**